### ISSUE TITLE
Order of navigation is not logical for "Refresh the application visual tree in inspector/ Select a UI element to inspect the running application/ Rotate left / Reset View / Rotate Right / Show frames / Show contents" in the 3D view of View Inspector window. 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#378]
### Steps to Reproduce

1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Create a Workbook with WPF Platform.
3. Run a sample workbook code.
4. Navigate to View Inspector tab and invoke it.
5. Now Navigate among 3D View of Inspector elements "Refresh the application visual tree in inspector/ Select a UI element to inspect the running application/ Rotate left / Reset View / Rotate Right / Show frames / Show contents" using tab .

### Expected Behavior
Order of navigation should be logical.
Expected Order of Navigation:
"Refresh the application visual tree in inspector ->Select a UI element to inspect the running application -> Rotate Left -> Reset View -> Rotate Right -> Show Frames -> Show contents"
### Actual Behavior
Tab order is not logical.
Actual Keyboard order is "Refresh the application visual tree in inspector -> Show Frames -> Show contents -> Select a UI element to inspect the running application -> Rotate Left -> Reset View -> Rotate Right "
### User Impact:
Keyboard user will be confused as the order of navigation is not meaningful.
### Screenshots:
![keyboard order 3d view](https://user-images.githubusercontent.com/22956130/40761271-7f8c64f6-64b8-11e8-8157-4c2493a7a3bd.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)
### Log File

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
